README.md

```diff
@@ -18,3 +21,49 @@
 
 ### [Week 9](weeks/week09/week09.md): Advanced AI-Driven Automation Techniques
 
+### [Week 10](weeks/week10/week10.md): Building a Test Automation Suite, utilizing both web and API testing
+
+
+## Fork the Repo
+
+A `fork` is a new repository that shares code and visibility settings with the original “upstream” repository. This is a good way to have your own repository and still having updates from the "upstream". Here the basic steps to take it. Follow the references to you want to get it deeper.
```

Comment:
Follow the references **if** you want to get it deeper.

Reply:
Fixed!